site/assets/js/notification.js

```diff
@@ -0,0 +1,60 @@
+import livestamp from "kuende-livestamp";
+import moment from "moment";
+
+$(document).ready(() => {
+    // Adds some styles to mark as seen button
+    const markAsSeenIcon = $(".markasseen-icon");
+
+    markAsSeenIcon.on('mouseover', function (e) {
+        e.preventDefault();
+        $(this).css("color", "#3eb0ac");
+    });
+    markAsSeenIcon.on('mouseleave', function (e) {
+        e.preventDefault();
+        $(this).css("color", "inherit");
+    });
+
+    // Mark a Notification as Seen on click
+    $(".notification-markasseen").on('click', function (e) {
+        e.preventDefault();
+        const notificationId = $(this).attr("data-notification");
+        const notifiableId = $(this).attr("data-notifiable");
+
+        $.ajax({
+            url: "/notification/seen",
+            data: {'notification': notificationId, 'notifiable': notifiableId},
+            method: 'post',
+            success: function (response) {
+                if (response === 'seen') {
+                    console.log('Marked notification as seen...');
+                    window.location.reload();
```

Comment:
*Either* you do client-side requests (then you need to hide/delete the items on the client side in addition to sending a request to the server), *or* you do normal form submissions (which reload the page, and then the list is freshly rendered with the remaining notifications). But don't do client-side JS and then reload the page.

Reply:
I did a client side request and hid the notification. However any insight on how to update the notification count on the side of the notification bell icon (which gets updated on reload)

Reply:
- Also, for form submits, we need to send a response to redirect the user to the page he was in currently when he clicks on the button, which we can't know as the dropdown is present in all pages. 
- Client-side request will not change the notification count immediately for which the user needs to refresh the page

Reply:
@imphil I fixed the notification dropdown part and have pushed the changes.
I think since notification inbox is a separate page we can use form submit here